packages/ava/src/advisor/ruler/rules/limit-series.ts

```diff
@@ -40,14 +40,15 @@ export const limitSeries: RuleModule = {
       if (nominalOrOrdinalFields.length >= 2) {
         const sortedFields = nominalOrOrdinalFields.sort(compare);
 
+        // TODO @Guangmingyoubei: 确认下这个注释还需要吗
```

Comment:
would you finish it before the release of v3? @GuangMingYouBei

Reply:
nope